lib/Middleware/ContentTypeMiddleware.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buzz\Middleware;
+
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\StreamInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ContentTypeMiddleware implements MiddlewareInterface
+{
+    /**
+     * Allow to disable the content type detection when stream is too large (as it can consume a lot of resource).
+     *
+     * @var bool
+     *
+     * true     skip the content type detection
+     * false    detect the content type (default value)
+     */
+    protected $skipDetection;
+
+    /**
+     * Determine the size stream limit for which the detection as to be skipped (default to 16Mb).
+     *
+     * @var int
+     */
+    protected $sizeLimit;
+
+    /**
+     * @param array $config {
+     *
+     *     @var bool $skip_detection True skip detection if stream size is bigger than $size_limit.
```

Comment:
Travis output:
--- Original
+++ New
@@ @@
      *
-     *     @var bool $skip_detection True skip detection if stream size is bigger than $size_limit.
+     *     @var bool $skip_detection true skip detection if stream size is bigger than $size_limit
      *     @var int $size_limit size stream limit for which the detection as to be skipped.
      * }
      */

Isn't in this line the period should be in?

Reply:
Oh. Sorry my bad. Travis suggest to remove it